src/sys/unix/random.cr

```diff
@@ -17,5 +17,15 @@ module Sys
         raise "Failed to access secure source to generate random bytes!"
       end
     end
+
+    def self.random_number : UInt32
```

Comment:
Basically the main problem I have with this PR is this method arbitrarily getting 4 bytes and being called just `random_number`.